recipes/TAMPA/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: TAMPA
+  version: {{ version }}
+
+source:
+  url: https://github.com/dkoslicki/TAMPA/archive/v{{ version }}.tar.gz
+  sha256: d5558cd419c8d46bdc958064cb97f963d1ea793866414c025906ec15033512ed
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+    - ete3
+    - seaborn
+    - pandas
+    - matplotlib
```

Comment:
Your recipe fails linking because of [this linting rule](https://bioconda.github.io/contributor/linting.html#uses-matplotlib).